aws-athena/ctas/reporting-res_report_summary.sql

```diff
@@ -199,174 +186,95 @@ class_modes AS (
-    WHERE cur = 'Y'
-        AND deactivat IS NULL
-    GROUP BY parid, taxyr
-    UNION ALL
-    SELECT
-        pin AS parid,
-        year AS taxyr,
-        char_yrblt AS yrblt,
-        char_building_sf AS total_bldg_sf
-    FROM {{ ref('default.vw_pin_condo_char') }}
-    WHERE NOT is_parking_space
-        AND NOT is_common_area
-),
-
---- AGGREGATE ---
-
 -- Here we aggregate stats on AV and characteristics for each reporting group
 -- By township, assessment_stage, and property group
 values_town_groups AS (
     SELECT
-        av.triad,
+        triad,
         'Town' AS geography_type,
-        av.property_group,
-        av.assessment_stage,
-        av.township_code AS geography_id,
-        av.year,
-        APPROX_PERCENTILE(av.total, 0.5) AS fmv_median,
+        property_group,
+        assessment_stage,
+        township_code AS geography_id,
+        year,
+        APPROX_PERCENTILE(total, 0.5) AS fmv_median,
         COUNT(*) AS pin_n,
-        APPROX_PERCENTILE(al.total_land_sf, 0.5) AS land_sf_median,
-        APPROX_PERCENTILE(ab.total_bldg_sf, 0.5) AS bldg_sf_median,
-        APPROX_PERCENTILE(ab.yrblt, 0.5) AS yrblt_median
-    FROM all_values AS av
-    LEFT JOIN aggregate_land AS al
-        ON av.parid = al.parid
-        AND av.year = al.taxyr
-    INNER JOIN chars AS ab
-        ON av.parid = ab.parid
-        AND av.year = ab.taxyr
+        APPROX_PERCENTILE(total_land_sf, 0.5) AS land_sf_median,
+        APPROX_PERCENTILE(total_bldg_sf, 0.5) AS bldg_sf_median,
+        APPROX_PERCENTILE(yrblt, 0.5) AS yrblt_median
+    FROM all_values
     GROUP BY
-        av.assessment_stage,
-        av.triad,
-        av.township_code,
-        av.year,
-        av.property_group
+        assessment_stage,
+        triad,
+        township_code,
+        year,
+        property_group
 ),
 
 -- By township and assessment stage
 values_town_no_groups AS (
     SELECT
-        av.triad,
+        triad,
         'Town' AS geography_type,
         'ALL REGRESSION' AS property_group,
-        av.assessment_stage,
-        av.township_code AS geography_id,
-        av.year,
-        APPROX_PERCENTILE(av.total, 0.5) AS fmv_median,
+        assessment_stage,
+        township_code AS geography_id,
+        year,
+        APPROX_PERCENTILE(total, 0.5) AS fmv_median,
         COUNT(*) AS pin_n,
-        APPROX_PERCENTILE(al.total_land_sf, 0.5) AS land_sf_median,
-        APPROX_PERCENTILE(ab.total_bldg_sf, 0.5) AS bldg_sf_median,
-        APPROX_PERCENTILE(ab.yrblt, 0.5) AS yrblt_median
-    FROM all_values AS av
-    LEFT JOIN aggregate_land AS al
-        ON av.parid = al.parid
-        AND av.year = al.taxyr
-    INNER JOIN chars AS ab
-        ON av.parid = ab.parid
-        AND av.year = ab.taxyr
+        APPROX_PERCENTILE(total_land_sf, 0.5) AS land_sf_median,
+        APPROX_PERCENTILE(total_bldg_sf, 0.5) AS bldg_sf_median,
+        APPROX_PERCENTILE(yrblt, 0.5) AS yrblt_median
+    FROM all_values
```

Comment:
**suggestion:** You could probably refactor this and the following CTEs to reduce duplication a bit using some jinja templating, but it's not strictly necessary.

Reply:
I think this would be a fun exercise since I haven't used jinja, but perhaps in another PR?